src/PrestaShopBundle/Resources/config/admin/services.yml

```diff
@@ -89,3 +92,18 @@ services:
     #UTILS
     prestashop.utils.zip_manager:
         class: PrestaShopBundle\Utils\ZipManager
+
+    #TRANSLATIONS
+    prestashop.translation.translations_factory:
+        class: PrestaShopBundle\Translation\TranslationsFactory
+
+    ps.translations_factory:
```

Comment:
same here, my idea is to have a service the shorter possible.

To be honest, if I could do it before the stable release, I will probably change every ``prestashop.*`` service to ``ps.*`` because I don't think our customers will use a lot of external bundles and if they did it, what is the probability to conflict with another ``ps.*`` ?